test/y2storage/package_handler_test.rb

```diff
@@ -135,4 +135,46 @@
       end
     end
   end
+
+  describe "#set_proposal_packages_for" do
```

Comment:
Actually, the correct notation here would be:

```ruby
describe ".set_proposal_packages_for" do
# ...
end
```

Since `set_proposal_packages_for` is a class method and not and instance one.

Reply:
OK, I'll change that.